gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -243,11 +250,38 @@ class GateConfig extends RedisHttpSessionConfiguration {
       .create(type)
   }
 
+  private <T> SelectableService createClientSelector(String serviceName, Class<T> type, OkHttpClient okHttpClient) {
+    Service service = serviceConfiguration.getService(serviceName)
+    if (service?.getBaseUrls() == null) {
```

Comment:
This could potentially also return an empty list and then it wouldn't get caught.